Source/monster.cpp

```diff
@@ -192,7 +192,7 @@ void InitMonsterTRN(CMonster &monst)
 	}
 }
 
-void InitMonster(Monster &monster, Direction rd, int mtype, Point position)
+void InitMonster(Monster &monster, Direction rd, size_t mtype, Point position)

```

Comment:
How about replacing `size_t mtype` with `size_t typeIndex`? There's also `_monster_id type` sprinkled around, and the two are very different.

Reply:
Tried cleaning up the various names a bit, let me know what you think :)

Reply:
LGTM.